python/examples/minimal_build/README.md

```diff
@@ -24,38 +24,42 @@ an eye toward maximum portability across different Linux distributions. This
 may help for contributors debugging build issues caused by their local
 environments.
 
-## Fedora 31
+## Ubuntu 18.04
 
 Build image:
 
 ```
-docker build -t arrow_fedora_minimal -f Dockerfile.fedora
+docker build -t arrow_ubuntu_minimal -f Dockerfile.ubuntu
 ```
 
 Build with conda or pip/virtualenv:
 
 ```
 # With pip/virtualenv
-docker run --rm -t -i -v $PWD:/io arrow_fedora_minimal /io/build_venv.sh
+docker run --rm -t -i -v $PWD:/io arrow_ubuntu_minimal /io/build_venv.sh
 
 # With conda
-docker run --rm -t -i -v $PWD:/io arrow_fedora_minimal /io/build_conda.sh
+docker run --rm -t -i -v $PWD:/io arrow_ubuntu_minimal /io/build_conda.sh
 ```
 
-## Ubuntu 18.04
+## Fedora 31
+
+Fedora uses podman instead of docker. In case of issues please first ensure
```

Comment:
Ok, but this seems to conflate two things:
- running a container on a Fedora _host_
- running a Fedora container

Perhaps the note about podman vs. docker should be relegated to a separate section?

Reply:
Good point. I will preare separate section about it.